docs/topics/runners/gradle.md

```diff
@@ -0,0 +1,539 @@
+> should be generated, so make sure Kotlin Gradle Plugin is applied as well, or configure source sets manually.
+>
+{type="note"} 
+
+> If you are using Dokka in a 
+> [precompiled script plugin](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:precompiled_plugins), 
+> you will have to add [Kotlin Gradle plugin](https://kotlinlang.org/docs/gradle.html) as a dependency in order for
+> it to work properly:
+>
+> <tabs group="build-script">
+> <tab title="Kotlin" group-key="kotlin">
+> 
+> ```kotlin
+> implementation(kotlin("gradle-plugin", "%kotlinVersion%"))
+> ```
+> 
+> </tab>
+> <tab title="Groovy" group-key="groovy">
+> 
+> ```groovy
+> implementation 'org.jetbrains.kotlin:kotlin-gradle-plugin:%kotlinVersion%'
+> ```
+> 
+> </tab>
+> </tabs>
+>
+{type="note"}
+
+
+### Legacy plugin application
+
+If you cannot use plugins DSL for some reason, you can use
+[the legacy method](https://docs.gradle.org/current/userguide/plugins.html#sec:old_plugin_application) of applying
+plugins.
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+buildscript {
+    dependencies {
+        classpath("org.jetbrains.dokka:dokka-gradle-plugin:%dokkaVersion%")
+    }
+}
+
+apply(plugin="org.jetbrains.dokka")
+```
+
+> Note that by applying Dokka this way, certain type-safe accessors will not be available in Kotlin DSL. 
+>
+{type="note"}
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+buildscript {
+    dependencies {
+        classpath 'org.jetbrains.dokka:dokka-gradle-plugin:%dokkaVersion%'
+    }
+}
+
+apply plugin: 'org.jetbrains.dokka'
+```
+
+</tab>
+</tabs>
+
+## Generating documentation
+
+Dokka's Gradle plugin comes with [HTML](html.md), [Markdown](markdown.md) and [Javadoc](javadoc.md) formats built in,
+and adds a number of tasks for generating documentation, both for [single](#single-project-builds) and [multi-project](#multi-project-builds) builds.
+
+### Single project builds
+
+Use the following tasks to build documentation for simple, single project applications and libraries:
+
+| **Task**       | **Description**                                                                     |
+|----------------|-------------------------------------------------------------------------------------|
+| `dokkaHtml`    | Generates documentation in [HTML](html.md) format.                                  |
+| `dokkaGfm`     | Generates documentation in [GitHub Flavored Markdown](markdown.md#gfm) format.      |
+| `dokkaJavadoc` | Generates documentation in [Javadoc](javadoc.md) format.                            |
+| `dokkaJekyll`  | Generates documentation in [Jekyll compatible Markdown](markdown.md#jekyll) format. |
+
+By default, you will find generated documentation under `build/dokka/{format}` directory of your project.
+Output location, among other things, can be [configured](#configuration) separately.
+
+### Multi-project builds
```

Comment:
I debated myself on the use of `Multi-project` vs `Multimodule` as there can be confusion.

Decided to go with `Multi-project` since this is a topic about Gradle, and Gradle calls it [multi-project](https://docs.gradle.org/current/userguide/multi_project_builds.html#sec:creating_multi_project_builds) and all "modules" are really subprojects (even in buildscripts: `subprojects {}`)

Not sure if it's worth addressing naming in a separate note.